README.md

```diff
@@ -54,16 +54,6 @@ A: Existing formats can continue to be a proving ground for technologies, as nee
 * June ??? v1.0.0
   * Release initial version of spec with two independent non-experimental implementations from OCI members
 
-## Maintainers
```

Comment:
fair. While copying over the contributing and git commit style, I was thinking the README's should get tidied up.